src/main/scala/strawman/collection/View.scala

```diff
@@ -290,22 +287,107 @@ trait ArrayLike[+A] extends Any {
   def apply(i: Int): A
 }
 
+trait SeqView[+A] extends View[A] with SeqOps[A, SeqView, SeqView[A]] { self =>
+  override def view: SeqView[A] = this
+
+  override def toSeq = immutable.IndexedSeq.from(this)
```

Comment:
In general I prefer omitting the `override` keyword when we only _implement_ a member.

Reply:
I would use the default `Seq` here rather than the default `IndexedSeq`